pkgs/by-name/cr/crabfit-frontend/package.nix

```diff
@@ -0,0 +1,109 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  fetchYarnDeps,
+  nodejs,
+  yarn,
+  fixup_yarn_lock,
+  google-fonts,
+  api_url ? "http://127.0.0.1:3000",
+  frontend_url ? "crab.fit",
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "crabfit-frontend";
+  version = "0-unstable-2023-08-02";
+
+  src = fetchFromGitHub {
+    owner = "GRA0007";
+    repo = "crab.fit";
+    rev = "628f9eefc300bf1ed3d6cc3323332c2ed9b8a350";
+    hash = "sha256-jy8BrJSHukRenPbZHw4nPx3cSi7E2GSg//WOXDh90mY=";
+  };
+
+  sourceRoot = "source/frontend";
+
+  patches = [
+    ./01-localfont.patch
+    ./02-standalone-app.patch
+    ./03-frontend-url.patch
+  ];
```

Comment:
You should make a effort of up streaming the patches and link to the related PR's here so we know when we can delete the patches from nixpkgs. If upstreaming is not possible for whatever reason please add a short comment describing what each patch does.

Reply:
:+1: every patch should have an issue / PR to upstream.

Reply:
(ideally, we should even pull them from upstream.)

Reply:
I opened PRs for the standalone-app and listening-address patches, I'm not sure about the one for using a local version of the karla, as it is patched the same way in other packages. For the frontend url, it is a hacky way to do it and I don't want to upstream this mess, it should be possible to use interpolation (c.f. https://www.i18next.com/translation-function/interpolation ) to do it correctly but I don't have the time to do it right now

Reply:
Done for the patches having a PR